beacon-node-oapi.yaml

```diff
@@ -90,6 +92,8 @@ paths:
     $ref: "./apis/beacon/blocks/root.yaml"
   /eth/v1/beacon/blocks/{block_id}/attestations:
     $ref: "./apis/beacon/blocks/attestations.yaml"
+  /eth/v1/beacon/blocks/{block_id}/sync_committee_rewards:
```

Comment:
lets change this to `/eth/v1/beacon/rewards/sync_committee/{block_id}`

Reply:
Also changed the endpoint and folder structure, as in https://github.com/ethereum/beacon-APIs/pull/260